<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#04395e;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="0,0,85">
      <item>
       <layout class="QVBoxLayout" name="menuLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="topMargin">
         <number>100</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: #004e89;
color:#FFFAFB;
font:bold;
padding: 0 5px;
border-radius:10px;
</string>
          </property>
          <property name="text">
           <string>HR-MS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dashboardButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
text-align: left
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}
</string>
          </property>
          <property name="text">
           <string>Dashboard</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/resources/images/dashboard-svgrepo-com.svg</normaloff>:/images/resources/images/dashboard-svgrepo-com.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="organizationButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
text-align:left;
margin-right:15px;
height: 25px;
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
          </property>
          <property name="text">
           <string>Organizations</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/resources/images/company-svgrepo-com.svg</normaloff>:/images/resources/images/company-svgrepo-com.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="employeeButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
text-align:left
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
          </property>
          <property name="text">
           <string>Employees</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/resources/images/employee-contact-svgrepo-com.svg</normaloff>:/images/resources/images/employee-contact-svgrepo-com.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="salariesButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
text-align:left
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
          </property>
          <property name="text">
           <string>Salaries</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/resources/images/dollar-coin-svgrepo-com.svg</normaloff>:/images/resources/images/dollar-coin-svgrepo-com.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>29</width>
            <height>29</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="attendanceButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
text-align:left
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
          </property>
          <property name="text">
           <string>Attendance</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/resources/images/fingerprint-svgrepo-com.svg</normaloff>:/images/resources/images/fingerprint-svgrepo-com.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loanButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
text-align:left
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
          </property>
          <property name="text">
           <string>Loans</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/resources/images/calculator-desmos-svgrepo-com.svg</normaloff>:/images/resources/images/calculator-desmos-svgrepo-com.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">QFrame[frameShape=&quot;4&quot;] /* QFrame::HLine == 0x0004 */
{
    color: red;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="bodyLayout" stretch="0,0,90">
        <item>
         <layout class="QHBoxLayout" name="titlebarLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="styleSheet">
             <string notr="true">background-color:#FFFAFB;
border-radius:10px;
width:250px;
height: 25px;
margin: 3px 0;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QStackedWidget" name="stackedWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="employeeScreen">
              <widget class="QWidget" name="verticalLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>50</y>
                 <width>312</width>
                 <height>80</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="styleSheet">
                   <string notr="true">font-size: 20px;
background-color: #004e89;
color:#FFFAFB;
font:bold;
padding: 0 5px;
border-radius:20px;
</string>
                  </property>
                  <property name="text">
                   <string>Our Successful Employees: 670</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="verticalLayoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>150</y>
                 <width>321</width>
                 <height>80</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="styleSheet">
                   <string notr="true">font-size: 20px;
background-color: #004e89;
color:#FFFAFB;
font:bold;
padding: 14px;
border-radius:20px;
</string>
                  </property>
                  <property name="text">
                   <string>Trusteb by 27+ Organizations</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="verticalLayoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>359</x>
                 <y>59</y>
                 <width>758</width>
                 <height>171</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="styleSheet">
                     <string notr="true">font-size: 20px;
background-color: #004e89;
color:#FFFAFB;
font:bold;
padding: 0 5px;
border-radius:10px;
</string>
                    </property>
                    <property name="text">
                     <string>For More Queries , contact us at +923001234567 or Email is at HR@Hrms.com</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="verticalLayoutWidget_4">
               <property name="geometry">
                <rect>
                 <x>29</x>
                 <y>279</y>
                 <width>985</width>
                 <height>301</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="styleSheet">
                     <string notr="true">font-size: 20px;
background-color: #004e89;
color:#FFFAFB;
font:bold;
padding: 0 5px;
border-radius:10px;
</string>
                    </property>
                    <property name="text">
                     <string>Our employess salary graph:    setting kr lein apai ty ethy graph pa la</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="QWidget" name="page_2">
              <widget class="QWidget" name="gridLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>221</width>
                 <height>101</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout"/>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
